### DISCUSSION TITLE
Gracenote

### DISCUSSION THREAD

**Discussion by djmarkino94** [Mr-MIBonk/M.I.B._More-Incredible-Bash#71] [General]
hi all, one question, what improvements does gracenote update bring?  I installed mib and everything works great.  I would like to understand what the gracenote is for and how to update it.  a thousand thanks

**Comment by harman-f** [comment]
It will just update the DB for media metadata on your unit (offline)

https://en.wikipedia.org/wiki/Gracenote

No changed to functions or FW.

**Reply by djmarkino94** [reply]
ok but what do you mean by metadata?  can the radio logos be updated?  do I have to enter the green menu to update?

**Comment by neoalius** [comment]
I have the gracenote version 12 file ,,,can some one please give the procedure to update

**Reply by djskual** [reply]
> I have the gracenote version 12 file ,,,can some one please give the procedure to update

Think you just need to get the lastest version on [https://mibsolution.one/#/1/9/MHI2%20-%20HARMAN/Firmware](https://mibsolution.one/#/1/9/MHI2%20-%20HARMAN/Firmware)
Put on your sd card and introduce in your MIB.
It's how i done with Gracenote V22.

**Reply by cirrussr20** [reply]
Hello - I have a problem with Gracenote V21 and V22 downloads. The SD card is recognised with V21 or V22 but fails at 35%, I have tried several cards always the same. Navdb uploads fine. Can I delete the Gracenote partition or restore some other way.

**Comment by djskual** [comment]
Have you gracenote 2 or just gracenote?

I installed gracenote 2 paylaod with firmware update, then updated to v22

**Comment by cirrussr20** [comment]
It's Gracenote 2 - I am at latest firmware so cannot update? I have no idea why it fails at 35% - I wonder if there is corruption so need to erase the directory/partition, I am guessing????

**Reply by djskual** [reply]
> It's Gracenote 2 - I am at latest firmware so cannot update? I have no idea why it fails at 35% - I wonder if there is corruption so need to erase the directory/partition, I am guessing????

Sorry, but i dont know more. I m just a casual user....

**Comment by neoalius** [comment]
MIB Unit Details: MU1438-MHI2_ER_AUG22_K3346, GrateNote version = 22, Audi A4 B9 2017.

Is it possible to get Gracenote displayed in VC from CARPLAY Spotify. 
I am getting Gracenote in VC when from Spotify when playing over BlueTooth or USB .

**Comment by RamisA4q** [comment]
[Neoalius](https://github.com/neoalius) Hi. Would you have the MU1438-MHI2_ER_AUG22_K3346 download file, mibsolution website below and I need all the batches for the update, can you share them via one drive or something? I would be very grateful for the files

**Comment by Eazydhub** [comment]
Hi 

I have a Porsche Macan 2017 and getting this error when trying to install the latest AIO version.

Device reports error - Gracenote2 Database:0 ErrorCode 134

Any ideas?
